docs/fields-configuration.md

```diff
@@ -11,10 +11,11 @@ The config file is a yaml file consisting of root level `fields` object that's a
 For each config entry the following fields are available:
 - `name` *mandatory*: dotted path field, matching an entry in [Fields definition](./glossary.md#fields-definition)
 - `fuzziness` *optional (`long` and `double` type only)*: when generating data you could want generated values to change in a known interval. Fuzziness allow to specify the maximum delta a generated value can have from the previous value (for the same field), as a delta percentage; value must be between 0.0 and 1.0, where 0 is 0% and 1 is 100%. When not specified there is no constraint on the generated values, boundaries will be defined by the underlying field type
-- `range` *optional (`long` and `double` type only)*: value will be generated between `min` and `max`
+- `range` *optional (`long` and `double` type only)*: value will be generated between `min` and `max`, eventually according to the defined `fuzziness`.
```

Comment:
It is not clear to me what fuzziness means in this context?

Reply:
I will add an extra sentence.
Basically it means that every random value will be generated between min and max.
A specific value will be generated within a delta defined by fuzziness from the previous value, if fuzziness is defined. In any case (fuzziness or not) the value would not escape the min/max range.